tests/Mafin.Web.Api.Rest.Tests.Unit/Authentication/BasicAuthHandlerTests.cs

```diff
@@ -25,7 +25,7 @@ public async Task SendAsync_WhenPassedCredentials_ShouldSetHeader()
         };
 
         using TestBasicAuthHandler handler = new(userName, password);
-        _ = await handler.PublicSendAsync(requestMessage).ConfigureAwait(false);
+        _ = await handler.PublicSendAsync(requestMessage).ConfigureAwait(true);
```

Comment:
`ConfigureAwait` in tests project, so much noise.

Reply:
Fair enough, makes little sense adding them in tests.
Removed ConfigureAwait() from test project.

Also, added task to myself to suppress the suggestion for test projects via .editorconfig when updating it